lib/webworker/ImportScriptsChunkLoadingRuntimeModule.js

```diff
@@ -169,7 +169,9 @@ class ImportScriptsChunkLoadingRuntimeModule extends RuntimeModule {
 						Template.indent([
 							"var success = false;",
 							`${globalObject}[${JSON.stringify(
-								compilation.outputOptions.hotUpdateGlobal
+								typeof compilation.outputOptions.hotUpdateGlobal === "string"
+									? compilation.outputOptions.hotUpdateGlobal
+									: compilation.outputOptions.hotUpdateGlobal(chunk)
```

Comment:
Can we use `compilation.getPath(compilation.outputOptions.hotUpdateGlobal, { chunk, contentHashType: "javascript" })`?

Reply:
of course

Reply:
Hey @alexander-akait, I'm implementing this, but by doing that, I need to change the signature of the function everywhere if I'm understanding correctly right?

And do you expect this call `compilation.getPath(compilation.outputOptions.hotUpdateGlobal, { chunk, contentHashType: "javascript" })` in every place that the function `hotUpdateGlobal` is being called? for example in `lib/javascript/ArrayPushCallbackChunkFormatPlugin.js` is accessed through `runtimeTemplate` instead of the compilation object, is it expected to be changed there to so its called from the compilation? 

and the same thing in `lib/web/JsonpChunkLoadingRuntimeModule.js`? thank you!